core.lua

```diff
@@ -2623,7 +2623,7 @@ do
 			if type(link) ~= "string" then
 				return
 			end
-			local inst, lvl, a1, a2, a3 = link:match("keystone:(%d+):(%d+):(%d+):(%d+):(%d+)")
+			local inst, lvl, a1, a2, a3 = link:match("keystone:138019:(%d+):(%d+):(%d+):(%d+):(%d+)")
```

Comment:
There are actually two different keystone items now. I'm betting that they basically made a new "Keystone" item that will be what we get during BFA.  So I think we should change the regex to just take any integer there for the item id.

Current keystone item: https://www.wowhead.com/item=138019/mythic-keystone
New keystone item (?): https://www.wowhead.com/item=158923/mythic-keystone